SceneExplorer/ModEntryPoint.cs

```diff
@@ -1,35 +1,32 @@
-﻿
-
+﻿using Colossal.IO.AssetDatabase;
```

Comment:
It seems like it works correctly now, but I had very strange runtime issues with AssetDB `using` outside of `SceneExplorer` namespace in the past

Reply:
I'll do some investigation regarding this...

Reply:
I was happening when the mod was loading/game was enabling it, but I cannot reproduce it so 🤷‍♂️

Reply:
I did research it a little bit, and found the following:
- C# usings are compiler-only features, they have no runtime effects
- Their scope is the actual .cs file, so they cannot influence anything outside of it
- The only difference if a using is inside or outside of the local namespace is only the visibility of the "used" namespace

Based on this I cannot think how it caused issues with the AssetDB. 🤔

Anyway, it is not reproducible anymore, and it compiles successfully, so I assume it is fine...